### PR TITLE
Disable geocoding options in on-premise

### PR DESCRIPTION
**Pull request by Xatpy**
Ticket related #1227
Disable geocoding by (city names, admin. regions, postal codes and ip)
if the feature flag ‘on_premise' is enabled.
Also, I have deleted lib/assets/test/spec/test_runner.html because it
wasn’t being used.

Please, @javisantana @xavijam @viddo  could you review it? Thanks!

### CODE REVIEW

**Comment by Xatpy** [comment]
Please, @javisantana , could you review it? It's using new naming as we discussed ( https://github.com/CartoDB/cartodb-onpremise/issues/48 )

File: config/grunt_development.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "browserify_debug": true,
-  "browserify_watch": true,
+  "browserify_watch": false,
```
Comment (javisantana) [inline config/grunt_development.json]: not sure if this change should go here

File: lib/assets/javascripts/cartodb/table/geocoding_dialog/geocoding_dialog_content.js [hunk 2]
```diff
@@ -38,10 +38,11 @@
 
     _TEXTS: {
       limits: {
-        key:      _t('<%= name %> key is not specified and panel can\'t be enabled'),
-        account:  _t('<%= name %> data source is not available in your plan. Please upgrade'),
-        credits:  _t('You\'ve reached the available <%= name %> credits for your account this month'),
-        quota:    _t('Your geocoding quota is not defined, please contact us at support@cartodb.com')
+        key:             _t('<%= name %> key is not specified and panel can\'t be enabled'),
```
Comment (javisantana) [inline lib/assets/javascripts/cartodb/table/geocoding_dialog/geocoding_dialog_content.js]: say no to alignment

File: lib/assets/javascripts/cartodb/models/user.js [hunk 3]
```diff
@@ -78,6 +78,17 @@ cdb.admin.User = cdb.core.Model.extend({
     })
     this._fetchingTables = true;
     tables.fetch();
+  },
+
+  // Checks if the user has the feature flag passed by param.
+  // Return: true if the user has that feature flag enabled. False if not.
+  featureEnabled: function(name) {
```
Comment (javisantana) [inline lib/assets/javascripts/cartodb/models/user.js]: this needs a test

File: config/grunt_development.json (hunk 1 above)
Comment (viddo) [inline config/grunt_development.json]: This value matches [master](https://github.com/CartoDB/cartodb/blob/master/config/grunt_development.json) though, is this branch up-to-date?

Comment (Xatpy) [inline config/grunt_development.json]: I think that no, so I won't commit this file

File: lib/assets/javascripts/cartodb/table/geocoding_dialog/geocoding_dialog_content.js (hunk 2 above)
Comment (Xatpy) [inline lib/assets/javascripts/cartodb/table/geocoding_dialog/geocoding_dialog_content.js]: Sorry, I tried to write as it was ( https://github.com/CartoDB/cartodb/blob/master/lib/assets/javascripts/cartodb/table/geocoding_dialog/geocoding_dialog_content.js#L41-L44 ), but I'll remove my alignment.

**Comment by Xatpy** [comment]
@javisantana why did Jenkins fail? _SpecRunner passed correctly.

**Comment by javisantana** [comment]
@Xatpy looks like a problem with clinker itself: http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/1089/console

**Comment by Xatpy** [comment]
retest this please

**Comment by Xatpy** [comment]
@javisantana  I retested it, but same result.
Error: 
```
stderr: error: The following untracked working tree files would be overwritten by checkout:
10:20:31 	themes/css/cartodb.css
10:20:31 Please move or remove them before you can switch branches.
```
I don't understand it, because I didn't commit cartodb.css

Could be `config/grunt_development.json` the reason?

**Comment by javisantana** [comment]
@Xatpy dunno, but try to remove that change (you finally didn't do it)

**Comment by Xatpy** [comment]
retest this please

**Comment by Xatpy** [comment]
Mail sent to  support@clinkerhq.zendesk.com asking for details about this.

**Comment by Xatpy** [comment]
retest this please

**Comment by Xatpy** [comment]
I close this pr because it was done in [this other PR](https://github.com/CartoDB/cartodb/pull/1806)

The reason is because there were a lot of conflicts here, and I preferred to start in an updated branch.

**Pull request closed without merging**